src/cpu/aarch64/jit_generator.hpp

```diff
@@ -312,10 +327,14 @@ class jit_generator : public Xbyak_aarch64::CodeGenerator, public c_compatible {
         }
 
         Xbyak_aarch64::ZRegS z_tmp(vmm_ubound.getIdx());
+        Xbyak_aarch64::VReg4S v_tmp(vmm_ubound.getIdx());
         Xbyak_aarch64::WReg w_tmp(reg_tmp.getIdx());
         float saturation_ubound = types::max_value<float>(odt);
         mov_imm(w_tmp, float2int(saturation_ubound));
-        dup(z_tmp, w_tmp);
+        if (isSVE) /* SVE is available. */
+            dup(z_tmp, w_tmp);
```

Comment:
```suggestion
            dup(isSVE ? z_tmp : v_tmp, w_tmp);
```

Reply:
Since Xbyak and Xbyak_aarch64 are implemented differently, `dup(isSVE ? z_tmp : v_tmp, w_tmp)` will cause a compilation error. Thank you for the suggestion.